api/controllers/ConfigController.js

```diff
@@ -0,0 +1,117 @@
+/**
+ * The ConfigController handles the creation and management of SMS app
+ * configuration.
+ */
+
+'use strict';
+
+var ObjectID = require('sails-mongo/node_modules/mongodb').ObjectID;
+
+module.exports = {
+
+  /**
+   * Display campaigns that have configurations set.
+   */
+  list: function(req, res) {
+    YesNoConfig.find({}, function(err, results) {
+      if (err) {
+        return res.status(500).send(err);
+      }
+
+      return res.view('configList', {configs: results});
+    });
+  },
+
+  /**
+   * Display view to create a new campaign configuration.
+   */
+  createView: function(req, res) {
+    return res.view(
+      'configCampaign',
+      {
+        id: 'new',
+        config: null
+      }
+    );
+  },
+
+  /**
+   * Handle the submission request to create a new campaign configuration.
+   */
+  createSubmit: function(req, res) {
```

Comment:
I'm not sure if this is also a Sails convention or not, but Laravel (and I think Rails?) use `index`, `create` (the view for creating a new resource), `store` (storing a new resource), `show`, `edit` (the view for editing a new resource), `update` (saving an updated resource), and `destroy` for the resource controller methods for each HTTP verb.